tests/requirements.txt

```diff
@@ -40,9 +40,9 @@ tox>=3.0.0, !=3.16.1
 flake8>=3.9.2
 # pyflakes - MIT licence
 # pyflakes>=2.5.0
-# pep8 - ??? licence
+# pep8 - MIT licence
 pep8>=1.0
-# pytest - ??? licence
+# pytest - MIT licence
 pytest>=7.0, !=8.1.0
 # pytest-checkdocs - ??? licence
```

Comment:
@coderabbitai,
check pipy.org for what the actual license of the python module "pytest-checkdocs" is, and where I can find that officially documented.